### DISCUSSION TITLE
personal notification server

### DISCUSSION THREAD

**Discussion by bravery-only** [Ylianst/MeshCentral#5933] [General]
personal notification server without using Telegram and WhatsApp, Nntfy ... in MeshCebtral
is it possible?

**Comment by si458** [comment]
**you really need to start following the github templates!**

what do you mean exactally?

you go click on 'My Account', then click 'Notification Settings' and then enable different types of notifications
the default once they are ticked, is a pop up inside the meshcentral web page,

you can also click 'Enable Web Notifications' which is the HTML5 notifications that power chrome/firefox to get notifications when the web page isnt open but the browser is open

you can also enable SMTP to get email alerts too...

the is loads of different messaging/notification types listed here - https://ylianst.github.io/MeshCentral/messaging/
